### PR TITLE
fixed footer , fixed .main overflow

### PR DESCRIPTION
**Pull request by electro199**
fixed footer and main div overflow

### CODE REVIEW

**Comment by electro199** [comment]
what does above msg mean

File: newspaper/static/sign-up.css [hunk 1]
```diff
@@ -52,8 +52,9 @@ a {
 }
 
 .gmt-icon {
-    margin-left: 6rem;
-    margin-right: 6rem;
+    margin: 0 6rem;
```
Comment (ImmaHarry) [inline newspaper/static/sign-up.css:55]: Looks good

**Review by ImmaHarry — CHANGES_REQUESTED** [review]
`index.css`: I think you need to add back the flex-wrap to keep the image and form+header in same row, if it works without it, well and good.
`sign-up.css`: Looks good.
`layout.css`: How does it work if you have removed `width:100vw` and `bottom:0` & `left:0`?
Oh and revert back to the original `.gitignore`

**Comment by electro199** [comment]
Sure I'll do it

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,5 @@ build/
 *.egg-info/
 
 .idea/
+instance
```
Comment (OpenSourceSimon) [inline .gitignore:13]: The instance folder is already ignored

File: .gitignore [hunk 3]
```diff
@@ -10,3 +10,5 @@ build/
...
+config.py.template
```
Comment (OpenSourceSimon) [inline .gitignore:14]: This is a template, so it needs to be in a commit

**Review by OpenSourceSimon — CHANGES_REQUESTED** [review]

**Review by OpenSourceSimon — APPROVED** [review]

**Pull request merged**